src/mono/wasm/testassets/BlazorHostedApp/BlazorHosted.Client/Pages/Chat.razor

```diff
@@ -86,7 +86,7 @@
     {
         await DisposeHubConnection();
         // exit the client
-        await JSRuntime.InvokeVoidAsync("eval", "import('./dotnet.js').then(module => { module.dotnet; module.exit(0); });");
+        Environment.Exit(0);
```

Comment:
This does not produce a console message that we're waiting for to finish the task. 
https://github.com/dotnet/runtime/blob/ba13ef001d1f26c4de4741a74e70bd591cc901f2/src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/AppTestBase.cs#L107
The test will never finish. Previous approach with `await JSRuntime.InvokeVoidAsync("eval", "getDotnetRuntime(0).exit(0);");` worked better.

Reply:
Or we can artificially print the message before it but it's a workaround.

Reply:
We can add `dotnet.withExitCodeLogging()` and do manual blazor startup

Reply:
I added `withExitCodeLogging`